includes/task.h

```diff
@@ -0,0 +1,11 @@
+#ifndef TASK_H
+#define TASK_H
+
+#include <stdint.h>
+
+#define TASK_LIMIT 16
+
+void    task_add(void (*task_code)(void), uint16_t stack_size);
```

Comment:
uint16_t -> size_t

Reply:
Isn't it too big? We have only 32Kbytes of RAM

Reply:
it's going to be passed in 32-bit register anyway